manifests/realm.pp

```diff
@@ -115,30 +115,29 @@
 # Copyright 2013 Patrick Mooney.
 # Copyright (c) IN2P3 Computing Centre, IN2P3, CNRS
 #
-define mit_krb5::realm(
-  $kdc                    = '',
-  $master_kdc             = '',
-  $admin_server           = '',
-  $database_module        = '',
-  $default_domain         = '',
-  $v4_instance_convert    = '',
-  $v4_realm               = '',
-  $auth_to_local_names    = '',
-  $auth_to_local          = '',
-  $kpasswd_server         = '',
-  $v4_realm_convert       = '',
-  $pkinit_anchors         = '',
-  $pkinit_pool            = '',
-  Boolean $rotate_servers = false,
-  $http_anchors           = '',
+define mit_krb5::realm (
+  Optional[Array] $kdc                  = undef,
+  Optional[String] $master_kdc          = undef,
```

Comment:
I'm not that sure, but this may be better if it was `Array[String]` . `krb5.conf(5)` states that `primary_kdc` (the new name for `master_kdc`) parameter "_Identifies the primary KDC(s)_" . So multiple hosts should be expected / allowed :shrug: